cabal.project

```diff
@@ -1,14 +1,8 @@
 -- Bump this if you need newer packages
 index-state: 2022-02-22T20:47:03Z
 
-packages: marlowe
-          marlowe-actus
-          marlowe-playground-server
-          marlowe-dashboard-server
-          marlowe-symbolic
-          marlowe-cli
+packages: marlowe-playground-server
           marlowe-contracts
```

Comment:
Why is this here? Shouldn't it stay in marlowe-cardano?

Reply:
As I understand, those are here because the PlayGround builds them and use them as part of the example programs. But I'll see if we can get rid of it in the future.